src/main/kotlin/christmas/domain/Order.kt

```diff
@@ -0,0 +1,40 @@
+package christmas.domain
+
+import java.time.DayOfWeek
+import java.time.LocalDate
+
+class Order(
+    val menus: Map<Menu, Int>,
+    private val orderDate: Date
+) {
+
+    init {
+        require(menus.values.sumOf { it } <= MAX_ORDER_MENU_AMOUNT) {
+            "you can order $MAX_ORDER_MENU_AMOUNT menus at one time"
+        }
+    }
+
+    constructor(menus: Map<Menu, Int>, orderDate: Int) : this(menus, Date(orderDate))
+
+    fun getDayOfWeek(): DayOfWeek {
+        val decemberFirst = LocalDate.of(EVENT_YEAR, DECEMBER, DECEMBER_START_DATE)
+        return decemberFirst.plusDays((date - 1).toLong()).dayOfWeek
+    }
```

Comment:
```suggestion
    fun getDayOfWeek(): DayOfWeek {
        return LocalDate.of(EVENT_YEAR, DECEMBER, date).dayOfWeek
    }
```

This one should also work.

Reply:
wow amazing